pkgs/by-name/ma/managarr/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+}:
+
+let
+  pname = "managarr";
+  version = "0.4.2";
+in
+
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "Dark-Alex-17";
+    repo = pname;
```

Comment:
I don't think this is correct, if anything it'd be `repo = "managarr";` as a string, however currently I'm using the variable. Should it be changed to a fixed string?

Reply:
It should be string, I apologize for the typo. My point is we should not reuse `pname` anywhere else (unless it's a composite package requiring pname in its steps), as it would break the build if pname is overriden.

Reply:
Thanks for the explanation. I have changed it to the fixed string.

I have also included the update script and squashed all commits into one. Not sure setting the script explicitly is necessary, the information is a bit sparse.